sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -3691,6 +3691,22 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
       checkAnswer(sql("SELECT id FROM t WHERE (SELECT true)"), Row(0L))
     }
   }
+
+  test("SPARK-33233: CUBE/ROLLUP/GROUPING SETS support GROUP BY ordinal") {
```

Comment:
Please add tests in `SQLQueryTestSuite`.

Reply:
> Please add tests in `SQLQueryTestSuite`.

Updated and check with 
![image](https://user-images.githubusercontent.com/46485123/97177980-d8553a80-17d1-11eb-8a18-133e3326a84c.png)

result is same.